src/components/templates/AnimateMap/game/map/entities/createBotEntity.ts

```diff
@@ -82,7 +82,8 @@ export const createBotEntity = (
         () => {
           // remove tooltip
           entity.remove(TooltipComponent);
-        }
+        },
+        true
```

Comment:
```suggestion
        }
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2161/commits/470154cca18bb5273e216b7d43acaace4ac25b4a